en/rest/hooks.mdown

```diff
@@ -0,0 +1,607 @@
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{
+  "className": "Tournament",
+  "triggerName": "beforeDelete",
+  "url": "https://api.example.com/Tournament/beforeDelete",
+  "warning": "beforeDelete trigger already defined for class Tournament in cloud code"
+}
+```
+
+## Delete function webhook
+To delete a function webhook use the put method.
+
+```bash
+curl -X PUT \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  -d '{"__op": "Delete"}' \
+https://api.parse.com/1/hooks/functions/foo
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/functions/foo', json.dumps(
+      {"__op": "Delete"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{}
+```
+
+If a cloud code trigger with the same name already exists then the it is returned as the result.
+
+```bash
+curl -X PUT \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  -d '{ "__op": "Delete" }' \
+https://api.parse.com/1/hooks/functions/sendMessage
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/functions/sendMessage', json.dumps(
+      {"__op": "Delete"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{ "functionName": "sendMessage" }
+```
+
+## Delete trigger webhook
+To delete a trigger webhook use the put method.
+
+```bash
+curl -X PUT \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  -d '{ "__op": "Delete" }' \
+https://api.parse.com/1/hooks/triggers/Game/beforeSave
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/triggers/Game/beforeSave', json.dumps(
+      {"__op": "Delete"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{}
+```
+
+If a cloud code trigger with the same name already exists then the it is returned as the result.
```

Comment:
Also add something like "Since the overriding webhook was just deleted, this cloud code trigger will be run the next time a `Game` object is saved."